generators/typescript-mcp/server/src/readme/ReadmeGenerator.ts

```diff
@@ -0,0 +1,39 @@
+import { File } from "@fern-api/base-generator";
+import { RelativeFilePath, join } from "@fern-api/fs-utils";
+import { TypescriptCustomConfigSchema } from "@fern-api/typescript-ast";
+import { FileGenerator } from "@fern-api/typescript-mcp-base";
+
+import { ServerGeneratorContext } from "../ServerGeneratorContext";
+
+export class ReadmeGenerator extends FileGenerator<File, TypescriptCustomConfigSchema, ServerGeneratorContext> {
```

Comment:
We might want to add a comment here justifying why this is separate from the rest of the `README.md` generation.